scripts/components/OwnerRepairRequestStatus.jsx

```diff
@@ -0,0 +1,121 @@
+var React = require('react');
+var moment = require('moment');
+var Api = require('../utils/Api.js');
+var MuiContextified = require('./MuiContextified.jsx');
+var mui = require('material-ui');
+var SelectField = mui.SelectField;
+var RaisedButton = mui.RaisedButton;
+var Dialog = mui.Dialog;
+
+var OwnerRepairRequestStatus = React.createClass({
+  getInitialState() {
+    return {
+      repairStatus: '',
+    }
+  },
+  
+  propTypes: {
+    statusChanged: React.PropTypes.func,
+    propertyId: React.PropTypes.number,
+    requestId: React.PropTypes.number,
+  },
+
+  onButtonClick() {
+    this.refs.dialog.show();
+  },
+
+  // Update select menu on change.
+  onSelectChange(event, selectedUserTypeIndex) {
+    var repairStatus = status[selectedUserTypeIndex].name;
+    this.setState({ 
+      repairStatus: repairStatus,
+    });
+  },
+
+  // Handle the form submission event when the user updates repair request.
+  onSubmit(event) {
+
+    // API call to update repair request status
+    var propertyId = this.props.propertyId;
+    var requestId = this.props.requestId;
```

Comment:
Line 39, Line 40. Both defined but never used as you use `this.props.requestId` and `this.props.propertyId` in the `data` object literal.